### ISSUE TITLE
nbopen -p uses only default profile.

### ISSUE THREAD

**Issue by mforbes** [takluyver/nbopen#10]
When running ``nbopen -p my_profile Notebook.ipynb``, a new notebook server is started using ``profile_default`` rather than ``profile_my_profile``.  This is probably because [line 37 for nbopen.py](https://github.com/takluyver/nbopen/blob/master/nbopen.py#L37) does not pass the ``--profile`` flag:
```python
                                        argv=[],  # Avoid it seeing our own argv
```

Instead, it should probably be something like

```python
                                        argv=['--profile', profile],  # Avoid it seeing our own argv
```

It appears that  ``nbopen`` is careful to only share existing servers if they are running the same profile, so this is only an issue when starting a new nbserver.

**Comment by takluyver** [comment]
This is true, but as we're moving the notebook to the name Jupyter, we're planning to drop the notion of explicit profiles for the notebook server - you'll still be able to use multiple config directories with an environment variable like `JUPYTER_DIR`. So I don't want to add more support for profiles in nbopen now.
